Assets/Scripts/Scenes/InGame/Player/PlayerStatus.cs

```diff
@@ -0,0 +1,188 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UniRx;
+using System;
+
+/// <summary>
+/// �v���C���[�̃X�e�[�^�X���Ǘ�����N���X
+/// </summary>
+namespace Scenes.Ingame.Player
+{
+
+    public class PlayerStatus : MonoBehaviour
+    {
+        //�v���C���[�̃f�[�^�x�[�X(���u��)
+        private int _playerID = 0;
+        private int _healthBase = 100;
+        private int _staminaBase = 100;
+        private int _sanBase = 100;
+
+        //���݂̃X�e�[�^�X�̕ϐ��i����l�b�g���[�N���\��j
+        //�������ɂ��Ă�����̓f�[�^�x�[�X���Q�Ƃ��čs���悤�ɂ���B
+        [SerializeField] private IntReactiveProperty _health = new IntReactiveProperty(0);//HP.�[���ɂȂ�Ǝ��S
+        [SerializeField] private IntReactiveProperty _stamina = new IntReactiveProperty(0);//�X�^�~�i
+        [SerializeField] private IntReactiveProperty _san = new IntReactiveProperty(0);//SAN�l
+        [SerializeField] private BoolReactiveProperty _survive = new BoolReactiveProperty(true);//����.true�̂Ƃ��͐����Ă���
+        [SerializeField] private BoolReactiveProperty _bleeding = new BoolReactiveProperty(false);//�o�����.true�̂Ƃ��Ɏ��Ԍo�߂ő̗͂�����
+
+
+        //���ꂼ��̍w�Ǒ������J����B����Class��Subscribe�ł���B
+        public IObservable<int> OnPlayerHealthChange { get { return _health; } }//_health(�̗�)���ω������ۂɃC�x���g�����s
+        public IObservable<int> OnPlayerStaminaChange { get { return _stamina; } }//_stamina(�X�^�~�i)���ω������ۂɃC�x���g�����s
+        public IObservable<int> OnPlayerSanValueChange { get { return _san; } }//_san(SAN�l)���ω������ۂɃC�x���g�����s
+        public IObservable<bool> OnPlayerSurviveChange { get { return _survive; } }//_survive(����)���ω������ۂɃC�x���g�����s
+        public IObservable<bool> OnPlayerbleedingChange { get { return _bleeding; } }//_bleeding(�o�����)���ω������ۂɃC�x���g�����s
+
+
+        [SerializeField] private DisplayPlayerStatusManager _displayPlayerStatusManager;
+
+        private void Init()
+        {
+            _health.Value = _healthBase;
+            _stamina.Value = _staminaBase;
+            _san.Value = _sanBase;
+        }
+        // Start is called before the first frame update
+        void Start()
+        {
+            //�K�v�ȃR���|�[�l���g�̎擾
+            _displayPlayerStatusManager = GameObject.Find("DisplayPlayerStatusManager").GetComponent<DisplayPlayerStatusManager>();
```

Comment:
```suggestion
_displayPlayerStatusManager = FindObjectOfType<DisplayPlayerStatusManager>();
``` 
このままだとGameObjectの名前が変更された際に簡単にエラーを吐くので、このやり方の場合はFindObjectOfTypeを使う方が良いです。また、別のコメントで書きますが、UI系の場合はこちらからGetComponentするのではなく、向こうからGetComponentさせた方が良いです（詳しくはMVRPパターン）